### ISSUE TITLE
Update multiple worksheets at onces

### ISSUE THREAD

**Issue by piwonesien** [mrjono1/FastExcel#52]
We have the situation that we want to fill an excel template that has several worksheets. Is there a better way to do this instead of:

`
using (FastExcel.FastExcel fastExcel = new FastExcel.FastExcel(templateFile, outputFileTmp))
{
    fastExcel.Write(worksheet1, "sheet1", 1);
}
using (FastExcel.FastExcel fastExcel = new FastExcel.FastExcel(new FileInfo(outputPathTmp), outputFile))
{
    fastExcel.Write(worksheet2, "sheet2", 1);
}
File.Delete(outputPathTmp);
`

**Comment by mrjono1** [comment]
I haven't tested this but this should work

```
using (FastExcel.FastExcel fastExcel = new FastExcel.FastExcel(templateFile, outputFileTmp))
{
    fastExcel.Write(worksheet1, "sheet1", 1);
    fastExcel.Write(worksheet2, "sheet2", 1);
}
File.Delete(outputPathTmp);
```

**Comment by piwonesien** [comment]
@mrjono1 yeah, it would be nice if this solution would work (that was also my first idea), but I'll get an error then:

```
System.Exception: 'Could not copy template to output file path'

IOException: The file '{...}' already exists.
```

**Comment by TorstenL** [comment]
Currently you have to reopen the file as a workaround.

using (FastExcel.FastExcel fastExcel = new FastExcel.FastExcel(templateFile, outputFileTmp))
{
    fastExcel.Write(worksheet1, "sheet1", 1);
 }
using (FastExcel.FastExcel fastExcel = new FastExcel.FastExcel(outputFileTmp))
{
    fastExcel.Write(worksheet2, "sheet2", 1);
}

**Comment by ahmedwalid05** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Update multiple worksheets at onces](https://trello.com/c/FfJ1D7eN/127-update-multiple-worksheets-at-onces)
